src/cli/index.js

```diff
@@ -1,5 +1,5 @@
-export * from "./help"
-export * from "./list"
-export * from "./version"
-export * from "./spawn"
-export * from "./options"
+export {help} from "./help"
```

Comment:
Workaround for the `export * from "..."` issue in Babel 6. See https://phabricator.babeljs.io/T2763